crm_exception/tests/test_crm_exception.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import Command
+from odoo.exceptions import ValidationError
+
+from odoo.addons.base.tests.common import BaseCommon
+
+
+class TestCrmLeadProbability(BaseCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stage_new = cls.env.ref("crm.stage_lead1")
+        cls.stage_qualified = cls.env.ref("crm.stage_lead2")
+        cls.stage_proposition = cls.env.ref("crm.stage_lead3")
+        cls.stage_won = cls.env.ref("crm.stage_lead4")
+        cls.opportunity = cls.env.ref("crm.crm_case_13")
+        cls.crm_exception = cls.env.ref("crm_exception.crm_excep_no_partner")
+        cls.crm_exception.active = True
+        cls.partner_id = cls.env.ref("base.res_partner_2")
```

Comment:
Just a minor point. If we decided to use `BaseCommon`, we can just remove that and use from pre-define partner.
https://github.com/odoo/odoo/blob/c43454c58bf85a40d7b6b28071ca3a5e7f5d004c/odoo/addons/base/tests/common.py#L32
```suggestion
```